Settings.cs

```diff
@@ -88,7 +88,7 @@ public bool ShowRemainingTime
         // preserve deserialized value or null when no change
         if (ShowRemainingTime != value)
         {
-          _showRemainingTime = value;
+          _showRemainingTime = false;
```

Comment:
This would set the value to false everytime the user tries to change it, so please remove this change. The correct way is to change the get function.